dev/com.ibm.ws.webcontainer.security.admin/src/com/ibm/ws/webcontainer/security/admin/internal/WebAdminSecurityConfigImpl.java

```diff
@@ -308,6 +309,16 @@ public String getSameSiteCookie() {
             return sameSiteCookie;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public boolean getPartitionedCookie() {
+        WebAppSecurityConfig globalConfig = WebAppSecurityCollaboratorImpl.getGlobalWebAppSecurityConfig();
+        if (globalConfig != null)
+            return WebAppSecurityCollaboratorImpl.getGlobalWebAppSecurityConfig().getPartitionedCookie();
```

Comment:
```suggestion
            return globalConfig.getPartitionedCookie();
```
I know you have it how the other methods do it, but it seems silly to get the `globalConfig` variable and then not use it here.

Reply:
I understand